dev/releases/create_stable_branch.py

```diff
@@ -14,81 +14,107 @@
 # TODO: implement parts of the steps described in
 # <https://github.com/gap-system/gap-distribution/blob/master/DistributionUpdate/STABLE_BRANCH_CHECKLIST.md>
 
-from utils import *
+from utils import error, notice, warning, patchfile
+import utils
 
 import subprocess
+import sys
 
 # Insist on Python >= 3.6 for f-strings and other goodies
-if sys.version_info < (3,6):
+if sys.version_info < (3, 6):
     error("Python 3.6 or newer is required")
 
-notice("Checking prerequisites")
-verify_command_available("git")
-verify_git_repo()
-verify_git_clean()
 
-# TODO: verify that we are on `master`, and that we are up-to-date (`git pull`)
+def usage(name: str) -> None:
+    print(f"Usage: `{name} MINOR`  creates the branch `stable-4.MINOR`")
+    sys.exit(1)
 
-# TODO: verify that `./configure && make` were already run
 
-gap_minor_version = 12  # TODO: this should be an argument or so?
-gapversion = f"4.{gap_minor_version}"
-nextgapversion = f"4.{gap_minor_version+1}"
-stable_branch = "stable-" + gapversion # TODO: how to specify this? probably have the version as argument?
+def main(gap_minor_version_str: str) -> None:
+    gap_minor_version = int(gap_minor_version_str)
+    gapversion = f"4.{gap_minor_version}"
+    nextgapversion = f"4.{gap_minor_version+1}"
+    stable_branch = "stable-" + gapversion
 
-# TODO: error out if the branch already exists
+    notice("Checking prerequisites")
+    utils.verify_command_available("git")
+    utils.verify_git_repo()
+    utils.verify_git_clean()
 
-# TODO: Create a pair of labels for GitHub issues called backport-to-X.Y and backport-to-X.Y-DONE.
+    notice(f"Switching to master branch")
```

Comment:
```suggestion
    notice("Switching to master branch")
```
Nothing is formatted, so the "f" is superfluous.